Add logo in center field in live football games

mamamia5x opened the pull request:
I was tinkering around and came up with a way to show the logo. Here is how they currently look.
![image](https://github.com/zengm-games/zengm/assets/57536929/2733ec1b-c9ef-486c-814f-3dba958651a3)
![image](https://github.com/zengm-games/zengm/assets/57536929/2abe406d-e15c-46e4-8979-718777e16528)

I'm not that good with TS, so it took me a bit to figure things out. 
_Notes:_ 
* The all star game isn't implemented in this new play by play, so the logo isn't implemented in there yet.
* The home team logo is shown in the championship game, even if in a neutral spot.
* If a team does not have a logo, it will not display the logo.

Please let me know if you have any questions or if there is anything else I can do to this.

dumbmatter commented:
Very cool! I thought about doing this myself but I decided it might look too busy. I'll leave this PR open in case I reconsider. For now, I still have a ways to go on the basic functionality of the new feature.

mamamia5x commented:
It's looking good so far.
Just a quick idea, maybe there could be a way to change grass color. Whether it's from the actual team data (like having jersey colors + grass colors), or just different shades of green picked by a random number generator.

mamamia5x commented:
Just updated the code to fit the current build, just have to resolve some conflicts. I changed the image size to fit around 10 yards in the center.
![image](https://github.com/zengm-games/zengm/assets/57536929/3329da72-32d1-4f9a-a40c-37dfab707c26)
